elephant/run.sh

```diff
@@ -1,7 +1,25 @@
 #!/bin/bash
 
 echo
-echo "This file has not been configured yet."
+echo "Updating network channels..."
```

Comment:
can you first print a short description of what the update does, and then ask the user if they would like to run the program (y/n)